docs/source/actor-rhel7-to-rhel8.md

```diff
@@ -0,0 +1,332 @@
+
+```shell
+# snactor run --verbose --print-output CheckOSRelease
+2018-11-23 13:11:15.549 INFO     PID: 18126 leapp: Logging has been initialized
+2018-11-23 13:11:15.578 INFO     PID: 18126 leapp.repository.system_upgrade_el7toel8: A new repository 'system_upgrade_el7toel8' is initialized at /usr/share/leapp-repository/repositories/system_upgrade/el7toel8
+2018-11-23 13:11:15.617 INFO     PID: 18126 leapp.repository.common: A new repository 'common' is initialized at /usr/share/leapp-repository/repositories/common
+[
+  {
+    "stamp": "2019-04-30T13:12:05.706317Z",
+    "hostname": "leapp-20190429150826",
+    "actor": "check_os_release",
+    "topic": "report_topic",
+    "context": "0ac49430-1b29-4940-92bb-3e81da85f8af",
+    "phase": "NON-WORKFLOW-EXECUTION",
+    "message": {
+      "hash": "686355c648832f5399cffc5bd0e7e8312d2b77577cdeec94f674a2565090cf7f",
+      "data": "{\"audience\": [\"sysadmin\"], \"detail\": \"{\\\"summary\\\": \\\"Minimal supported OS version for upgrade process: 7.6\\\"}\", \"flags\": [\"inhibitor\"], \"renderers\": {\"html\": \"<h2 class=\\\"report-title\\\">{{ title }}</h2><p class=\\\"report-summary\\\">{{ summary }}</p>\", \"plaintext\": \"{{ title }}\\n{{ summary }}\"}, \"severity\": \"medium\", \"title\": \"Unsupported OS version\"}"
+    },
+    "type": "Report"
+  }
+]
+```
+
+To flush all saved messages from the repository database, run `snactor messages clear`.
+
+## Writing tests for an actor
+
+[Read the tutorial](https://leapp.readthedocs.io/en/latest/unit-testing.html) for writing an running unit and component tests
+
+## Best practices
+
+Read the best practices [document](https://github.com/oamg/leapp-repository/blob/master/docs/best-practices.md) and [Python guidelines](https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md)
+
+## Contributing actors to the Leapp project
+
+Currently all Leapp elements (i.e. actors, models, tags) are stored under a public [GitHub repository](https://github.com/oamg/leapp-repository)
+
+All new content that needs to be part of Leapp release distributed to all users should be proposed as a Pull Request in this repository.
+
+**Before submitting your work for review, make sure you have read and followed the contribution guidelines:**
+
+[Contribution guidelines for writing actors](https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md)
+
+This [pull request](https://github.com/oamg/leapp-repository/pull/186) gives a good example of both guidelines-driven actor implementation and thorough test coverage
+
+## Pull requests review processes
+
+Every pull request need to be reviewed by core-team members, for optimize this process please ensure your code is following some criteria:
```

Comment:
@pirat89, this is really bad place to put such information. This fragments the docimentation event more than it is now.
Why isn't [this](https://github.com/oamg/leapp-guidelines/blob/master/contributing-guidelines.rst) good enough? If you want to add or change something in the PR, the linked doc is the place, not in this guide.

Reply:
@bocekm  you are right. it sounds better to have it rather there and put here just the link to the doc.

Reply:
I added the link to the documentation that @bocekm mentioned

Reply:
I rather meant to remove this `## Pull requests review processes` section completely from this tutorial.

@pirat89 can compare the https://github.com/oamg/leapp-guidelines/blob/master/contributing-guidelines.rst with what he has suggested here and create a new pull request to change https://github.com/oamg/leapp-guidelines/blob/master/contributing-guidelines.rst if needed.